task2/main_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import "testing"
+
+func Test_miniMaxSum(t *testing.T) {
+	type args struct {
+		arr []int
+	}
+	tests := []struct {
+		name  string
+		args  args
+		want  int
+		want1 int
+	}{
+		{
+			name:  "s",
+			args:  args{[]int{1, 2, 3}},
+			want:  3,
+			want1: 5,
+		},
+	}
```

Comment:
please add more cases in the test, like:
`{5,5,5,5,5}`
`{1,2}`
`{8,7,6,5,5,4,3,1,2}`
empty slice `{}`